noxfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+from pathlib import Path
+
+import nox
+
+LINT_FILES = ("hacking/pr_labeler/label.py", "noxfile.py")
+PINNED = os.environ.get("PINNED", "true").lower() in {"1", "true"}
+nox.options.sessions = ("lint",)
+
+
+def install(session: nox.Session, *args, req: str, **kwargs):
+    arguments = ["-r", f"tests/{req}.in"]
+    if PINNED:
+        arguments.extend(("-c", f"tests/{req}.txt"))
```

Comment:
Plz replace this with a `PIP_CONSTRAINT` env var since it propagates to the PEP 517 build envs that might be used for building sdists.

Reply:
Fixed in latest force push